tests/test.py

```diff
@@ -1106,6 +1188,16 @@ def get_connector_api_connection_string(self, port=None):
             port = self.CONNECTOR_API_PORT_1
         return get_connector_connection_string(port)
 
+    def get_ocsp_server_connection_string(self, port=None):
+        if not port:
+            port = self.OCSP_SERVER_PORT
+        return 'http://127.0.0.1:{}'.format(port)
```

Comment:
same here, better to prefer localhost instead 127.0.0.1

Reply:
Done